x11-packages/emacs-x/build.sh

```diff
@@ -5,6 +5,7 @@ TERMUX_PKG_MAINTAINER="@termux"
 # Update both emacs and emacs-x to the same version in one PR.
 TERMUX_PKG_VERSION=30.1
 TERMUX_PKG_REVISION=1
+TERMUX_PKG_ALIGN_VERSION_WITH=emacs
```

Comment:
`vim-gtk` should get the same treatment to align with `vim`.

Reply:
Done.